### DISCUSSION TITLE
Discrepancy in ShopShippingProfileUpgrade Type Definition: Enum String vs. Response Integer

### DISCUSSION THREAD

**Discussion by exytab** [etsy/open-api#1200] [Q&A]
There is an inconsistency between the OpenAPI definition for the ShopShippingProfileUpgrade and the actual response from Etsy's API. This discrepancy is concerning the type field.

OpenAPI Definition:
```json
"type": {
    "type": "string",
    "description": "The type of the shipping upgrade. Domestic (0) or international (1).",
    "enum": ["0", "1"]
}
```
Actual API Response:
```json
"shipping_profile_upgrades": [
{
...
  "upgrade_name": "Economy",
  "type": 0,
  "rank": 0,
...
}
]
```

Please, Update the OpenAPI specification to reflect that the `type` is returned as an integer

**Comment by openapi-support** [comment]
Thank you for bringing this to our attention. The issue has been [fixed](https://github.com/etsy/open-api/releases/tag/3.0.0-general-release-2025-10-14)!
